### PR TITLE
Add Ryan Bergman's Story

### PR DESCRIPTION
**Pull request by ryber**
Hello, I ran across this on twitter and thought it was neat and so thought I would add to it. Note that I am a terrible speller and nobody other than myself has proof read this so I welcome feedback. Also if you think I should expand some part thats fine too. any feedback is welcome. I didn't see a lot of non-SV/tech companies on your existing list so maybe this adds some more variety?

Thanks
Ryan

### CODE REVIEW

File: src/markdown/stories/ryan-bergman.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+
+**_What two or three factors were most important in you reaching Staff? How have the companies you joined, your location, or your education impacted your path?_**
+
+There are really two sides for what folk are looking for at this level. You need to be very technically capible but you also need to be someone people want to work with. Folk are going to come to you with problems they need solutions for and they want someone pragmatic and creative and empethetic to their goals. They also want someone who understnds all the parts that are going to have to change. Not just some small part of it.
```
Comment (lethain) [inline src/markdown/stories/ryan-bergman.md:44]: typo "capible"

Comment (lethain) [reply src/markdown/stories/ryan-bergman.md:44]: also "understnds"

Comment (lethain) [reply src/markdown/stories/ryan-bergman.md:44]: and "empethetic"

File: src/markdown/stories/ryan-bergman.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+Education-wise, I took an alternative path to this position. As a kid with undiagnosed ADHD and dyscalculia I developed an adversarial relationship with math. While I loved computers and loved hacking at home. I believed that a CompSci major wasn't something I was going to be able to do. So I went into the arts with the idea that I could back into computers from graphic design. I actually ended up majoring in City Planning (also in the design college). and then durring the the first dot-com boom I took a job with a tech startup doing web design while waiting for my fiancée to graduate. I never went back to planning and slowly worked my way down the stack. Still, I find I use my major all the time. Lots of city planning is about politics and how to navigate different groups with different agendas and needs. Once you're a little higher up in the org this is absolutely neccessary. I also learned about things like long term project management, how to run a public meeting, and GIS systems which have helped while working on agronomic data.
```
Comment (lethain) [inline src/markdown/stories/ryan-bergman.md:46]: "durring"

File: src/markdown/stories/ryan-bergman.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+**_What’s your advice to people pursuing a Staff role?_**
+
+From a tech perspective, constantly reach outside of your comfort zone and your existing wheelhouses. I've been programming professionally for 20 years and at different times I've had Perl, ColdFusion, PHP, JavaScript, C#, Java, and Scala all be the primary world I lived in. Nothing is a hammer and nothing won't eventually be replaced (except C).
+
+Being a senior staff engineer means understanding both low level code and **systems of systems**. I recomend reading up on "systems thinking" to understand the later. Everything exists as a whole. Microservices don't exist in isolation, they are all part of giant distributed monoliths. Once you're at the staff engineering level you need to understand the deep relationships that every part of your org's tech stack has on each other and on the outside world!
```
Comment (lethain) [inline src/markdown/stories/ryan-bergman.md:52]: "recomend"

File: src/markdown/stories/ryan-bergman.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+Several years ago I was given ownership of the permissions subsystem called SOUP (Security, Organizations, Users, and Partnerships). It was not my favorite part of the system but apparently it got out that I had opinions about it. So I grabbed one other developer and we went to work at what was mostly a gigantic refactoring to get a model that was unified and easy to inspect and slice in the ways the various systems needed. In the end we replaced multiple different sub-systems including a custom RBAC and a LDAP repo with a single rdbms table. We also built out many support tools and reports that didn't exist yet. The one hard part was getting alignment to do the outside work of doing a large change to the permission systems. That means UX, documentation, training, marketing, more UX and user testing. These parts were far more challenging from the people side than the tech. I had to spend a lot of time with different product owners showing them the bennifit to their own systems and explaining that if they wanted feature x, it would cost significantly less (or even be possible), if we can just align a little work across these other teams.
```
Comment (lethain) [inline src/markdown/stories/ryan-bergman.md:73]: "bennifit"

**Comment by lethain** [comment]
@ryber -- thank you for writing this up, and I'd love to merge it in. As you point out, there is a major overemphasis of folks in SF/NYC and I'm really excited to have a wider set of experiences represented.

I flagged a few typos. If you want to fix, please do, otherwise let me know and I can go ahead and fix them myself when I do a final pass on rendering!

Once we merge, I'll deploy and it'll email out the mailing list the following morning at 9AM PT. Assuming you're able to respond to this comment and let me know on the typos (e.g. let me know if you want to fix or if I should go ahead), then I'd plan to merge Sunday night to go out Monday.

**Comment by ryber** [comment]
Oh jeeze 🤦 IDK why I wrote that in the markdown editor without a spell checker. I found some other typos and fixed them as well.

**Comment by lethain** [comment]
@ryber -- thank you for the edits! I merged and did a small set of formatting changes to address some very minor rendering issues and consistency with other posts. Since you were able to turn it around so quickly I have it up at https://staffeng.com/stories/ryan-bergman and email should go out at 9AM PT this morning :-)

Thanks again for sharing your story!

**Pull request merged**